contracts/WitnetRequestsBoard.sol

```diff
@@ -204,6 +206,12 @@ contract WitnetRequestsBoard is WitnetRequestsBoardInterface {
     external
     drNotIncluded(_id)
  {
+    // Ensures the dr.epoch is smaller than the last epoch reported to the block relay
+    uint256 witnetEpoch = blockRelay.getLastEpoch();
+    require(requests[_id].epoch < witnetEpoch, "the current epoch is not compatible with the dr epoch");
```

Comment:
What's this check for? I think the `witnetEpoch` can never be lower than `requests[_id].epoch` and therefore this requirement is always met unless both terms are equal. I wonder what we are trying to prevent here.

Reply:
Yes, my fault. The check should be between the `dr.epoch` and the input `_epoch`. It's just been changed.